src/server/api/routes/retrieve-first-question.router.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const firstQuestionController = require('../controllers/retrieve-first-question.controller');
+
+/**
+ * @swagger
+ * /getFirstQuestion/{ID}:
```

Comment:
It's a matter of preferences, but it's redundant to name it "getFirstQuestion" plus the ID as a param. 
I am assuming that after the first question, it will come the second and so on, so I would think about this path like **question/1** (question ID = 1). But I might be missing something on the logic...

Reply:
Changed the naming appropriately and also the logic